test/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Xaptum, Inc.
+# Copyright 2020 Xaptum, Inc.
```

Comment:
Nit pick: use a range here to document the age of the project:
```
# Copyright 2017-2020 Xaptum, Inc.
```

Reply:
Yes, good catch